### ISSUE TITLE
[@review UI] Filter by base branch

### ISSUE THREAD

**Issue by eleven4y** [isachivka/review#28]
На странице `Pull Requests` добавить input (или select) с выбором base branch для фильтрации ПР

**Comment by isachivka** [comment]
@eleven4y привет!

Это очень хорошая идея, у тебя есть время на имплементацию?

**Comment by eleven4y** [comment]
да, я проработаю этот вариант в свободное время и вернусь с решением 
естимейты пока ставить не буду

**Comment by isachivka** [comment]
Будет очень круто! Я добавлю тебя на страничку "почетных контрибьюторов" если это выгорит
